services/UserService.js

```diff
@@ -375,3 +378,26 @@ export default class UserService {
     });
   }
 }
+
+
+/**
+ * User log out.
+ */
+export const logout = () => CTAPI.get('/auth/logout', {
+  headers: {
+    ...CTAPI.defaults.headers,
+    // TO-DO: forces the API to not cache, this should be removed at some point
+    'Upgrade-Insecure-Requests': 1
+  }
+}).then((response) => {
+  const { status, statusText } = response;
+  window.location.href = `/logout?callbackUrl=${window.location.href}`;
```

Comment:
This is part of your callback. Think about it: if I want to call this function in another side, with your code, no matter what, is going to redirect to that URL. 

I would move the redirect as part of the callback and return here just the `response` as there's no data to return.